### DISCUSSION TITLE
Menu bar

### DISCUSSION THREAD

**Discussion by DutchPete** [notaapp/talk#416] [Questions]
I am on macOS, and the menu bar is hidden unless you hover over it. Is there a way to have is displayed permanently? I have not been able to find a setting.

**Comment by astoilkov** [comment]
Are you talking about a menu bar inside of Nota or the macOS, OS level, menu bar?

**Comment by DutchPete** [comment]
I am talking about the black menu bar in the screenshot below. It is hidden until hovered over.

![image](https://github.com/notaapp/talk/assets/10794265/d532937f-30c1-4cc1-9e74-ebca9fd7d527)

**Reply by astoilkov** [reply]
That's a macOS settting in Control Center → Automatically hide and show the menu bar.
![CleanShot 2024-01-03 at 14 59 48@2x](https://github.com/notaapp/talk/assets/884810/9d36db0b-4c04-47f2-b801-a65b214d32a7)

**Comment by DutchPete** [comment]
I had it set to "in full screen only", which means Nota's standard display is Full Screen because this did not happen with any other apps.

Changing it to "Never" keeps the menu bar displayed indeed.

**Reply by astoilkov** [reply]
Maybe you put Nota in full screen by accident? Can you go to View → Toggle Fullscreen?

**Reply by DutchPete** [reply]
Yes, you are right, I must have done that by accident while experimenting with Nota.
Thanks for your patience 😀 👍
